core/arch/x86/decode_table.c

```diff
@@ -1430,12 +1508,12 @@ const instr_info_t * const op_instr[] =
 #define Zb  TYPE_XLAT, OPSZ_1
 #define Bq  TYPE_MASKMOVQ, OPSZ_8
 #define Bdq  TYPE_MASKMOVQ, OPSZ_16
-#define Kw  TYPE_FLOATMEM, OPSZ_2
-#define Kd  TYPE_FLOATMEM, OPSZ_4
-#define Kq  TYPE_FLOATMEM, OPSZ_8
-#define Kx  TYPE_FLOATMEM, OPSZ_10
-#define Ky  TYPE_FLOATMEM, OPSZ_28_short14 /* _14_ if data16 */
-#define Kz  TYPE_FLOATMEM, OPSZ_108_short94 /* _98_ if data16 */
+#define Fw  TYPE_FLOATMEM, OPSZ_2
```

Comment:
I think this rename should be a separate PR.

Reply:
Done: https://github.com/DynamoRIO/dynamorio/pull/3572